src/components/SearchModal/ManageLists.tsx

```diff
@@ -171,7 +171,7 @@ const ListRow = memo(function ListRow({ listUrl }: { listUrl: string }) {
 
   const { label } = CHAIN_INFO[chainId ? chainId : SupportedChainId.MAINNET]
   const plural = activeTokensOnThisChain === 1 ? '' : 's'
-  const hoverText = t`This token list currently has ${activeTokensOnThisChain} ${label} token${plural}`
+  const hoverText = t`This token list has ${activeTokensOnThisChain} ${label} token${plural}`
```

Comment:
this is not really going to work for non english unfortunately :(

That's because the string that will be extracted by the i18n library will look like `This token list has {0} {1} token{2}`. At run time we will try to add an 's' at position `{2}`, but languages have different ways of localizing plurals.

I think I'd rather use what you had before for this PR, not a big deal.